receiver/hostmetricsreceiver/testdata/e2e/expected_process.yaml

```diff
@@ -12,16 +12,16 @@ resourceMetrics:
             stringValue: sleep
         - key: process.executable.path
           value:
-            stringValue: /bin/sleep
+            stringValue: /usr/bin/sleep
         - key: process.owner
           value:
-            stringValue: atoulme
+            stringValue: neck
```

Comment:
If we're ignoring these attributes now anyway maybe these can be removed? Or does the integration test framework expect them in some way?

Reply:
Good point, the test framework actually removes them, so let's remove them as well https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36847/commits/3b5936a8c28d081b0703eccbe2716e19012ba018